test/utilities/apexUtils.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  TextEditor
+} from 'wdio-vscode-service';
+import {
+  runCommandFromCommandPrompt
+} from './commandPrompt';
+import {
+  pause
+} from './miscellaneous';
+
+export async function createApexClassWithTest(name: string): Promise<void> {
+  const workbench = await browser.getWorkbench();
+  let textEditor: TextEditor;
+
+  // Using the Command palette, run SFDX: Create Apex Class to create the main class
+  const inputBox = await runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+  // Set the name of the new Apex Class
+  await inputBox.setText(name);
+  await inputBox.confirm();
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+  await pause(1);
+
+  // Modify class content
+  const editorView = workbench.getEditorView();
+  textEditor = await editorView.openEditor(name + '.cls') as TextEditor;
+  const classText = [
+    `public with sharing class ${name} {`,
+    '\tpublic static void SayHello(string name){',
+    '\t\tSystem.debug(\'Hello, \' + name + \'!\');',
+    '\t}',
+    '}'
+  ].join('\n');
+  await textEditor.setText(classText);
+  await textEditor.save();
+  await textEditor.toggleBreakpoint(3);
+  await pause(1);
+
+  // Using the Command palette, run SFDX: Create Apex Class to create the Test
+  await runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+  // Set the name of the new Apex Class Test
+  await inputBox.setText(name + 'Test');
+  await inputBox.confirm();
+
+  // Select the default directory (press Enter/Return).
+  await inputBox.confirm();
+  await pause(1);
+
+  // Modify class content
+  textEditor = await editorView.openEditor(name + 'Test.cls') as TextEditor;
+  const testText = [
+    '@IsTest',
```

Comment:
Same comment about mixing string types.  Please change to using a back tick for all strings here.